cloudflare_cache/README.md

```diff
@@ -0,0 +1,29 @@
+# Cloudflare Cache #
+
+This example demonstrates how to purge Cloudflare cache when your live site's cache is cleared.
+
+## Instructions ##
+
+- Copy `cloudflare_cache.json` to `files/private` of the *live* environment after updating it with your cloudflare info.
+ - API key can be found in the `My Settings` page on the Cloudflare site.
+ - I couldn't find zone id in the UI. I viewed page source on the overview page and found it printed in JavaScript.
```

Comment:
"If you cannot find the zone ID in your Cloudflare settings page, then view the source on the overview page and find it in the embedded JavaScript code."